client/state/plans/selectors.js

```diff
@@ -10,7 +10,9 @@ import { get, find } from 'lodash';
  * Internal Dependencies
  */
 import createSelector from 'lib/create-selector';
-import { calculateMonthlyPriceForPlan } from 'lib/plans';
+import { calculateMonthlyPriceForPlan, getPlan as libGetPlan } from 'lib/plans';
```

Comment:
Do we have a more appropriate name for `libGetPlan`? Perhaps `getRawPlan` or something?